<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ffmpegui</class>
 <widget class="QMainWindow" name="ffmpegui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>391</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ffmpegui</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>581</width>
      <height>241</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Options</string>
     </attribute>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>2</x>
        <y>13</y>
        <width>571</width>
        <height>191</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Video</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="resolution">
               <property name="text">
                <string>Resolution</string>
               </property>
               <property name="buddy">
                <cstring>comboResolution</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboResolution"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="framerate">
               <property name="text">
                <string>Framerate</string>
               </property>
               <property name="buddy">
                <cstring>comboFramerate</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboFramerate"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="vbitrate">
               <property name="text">
                <string>Bitrate</string>
               </property>
               <property name="buddy">
                <cstring>comboBitrate</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBitrate"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Audio</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QLabel" name="samplerate">
               <property name="text">
                <string>Sample Rate</string>
               </property>
               <property name="buddy">
                <cstring>comboSamplerate</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboSamplerate"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="bitdepth">
               <property name="text">
                <string>Bit Depth</string>
               </property>
               <property name="buddy">
                <cstring>comboBitdepth</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBitdepth"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="abitrate">
               <property name="text">
                <string>Bitrate</string>
               </property>
               <property name="buddy">
                <cstring>comboAbitrate</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboAbitrate"/>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Output</string>
     </attribute>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>581</width>
        <height>211</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>22</y>
      <width>591</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pBSourcePath">
           <property name="text">
            <string>&amp;Source</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pBDestPath">
           <property name="text">
            <string>&amp;Destination</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLineEdit" name="sourcePath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="destPath">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="screencast">
           <property name="text">
            <string>Screencast</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="android">
           <property name="text">
            <string>Android</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="psp">
           <property name="text">
            <string>PSP</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>&amp;Convert -&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pBSourcePath</sender>
   <signal>clicked()</signal>
   <receiver>ffmpegui</receiver>
   <slot>setSource()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBDestPath</sender>
   <signal>clicked()</signal>
   <receiver>ffmpegui</receiver>
   <slot>setDestination()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>107</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ffmpegui</receiver>
   <slot>executeCommand()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>screencast</sender>
   <signal>clicked()</signal>
   <receiver>ffmpegui</receiver>
   <slot>setScreencast()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>60</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>128</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>android</sender>
   <signal>clicked()</signal>
   <receiver>ffmpegui</receiver>
   <slot>setAndroid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>psp</sender>
   <signal>clicked()</signal>
   <receiver>ffmpegui</receiver>
   <slot>setPSP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>254</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setSource()</slot>
  <slot>setDestination()</slot>
  <slot>executeCommand()</slot>
  <slot>setScreencast()</slot>
  <slot>setAndroid()</slot>
  <slot>setPSP()</slot>
 </slots>
</ui>
